spec/rails_helper.rb

```diff
@@ -1,4 +1,5 @@
 ENV["RAILS_ENV"] ||= "test"
+ENV["ORGANISATION_FILE_PATH"] = "spec/support/organisation_types.yml"
```

Comment:
Shouldn't this be in the `.env.test` file? And also set in `.env.development` for local development?